extensions/cornerstone-dicom-rt/src/viewports/OHIFCornerstoneRTViewport.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useCallback, useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { ViewportActionArrows } from '@ohif/ui';
 import { useViewportGrid } from '@ohif/ui-next';
+import { onSegmentChange as handleSegmentChange } from '../../../cornerstone/src/utils/segmentUtils';
```

Comment:
we shouldn't import like this, export it at the cornerstone as a utils, and then here do 

import { utils } from '@ohif/extension-cornerstone'